docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -3,112 +3,256 @@ sidebar_position: 2
 ---
 
 import FeedbackComponent from "@site/src/pages/feedback.md";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-# Generating SessionSigs
+# Generating SessionSigs: `getSessionSigs`
 
-You can use any wallet or auth method to generate session signatures with the `getSessionSigs()` function from the Lit SDK. This function generates a session keypair and uses a callback function that signs the generated session key to create an `AuthSig` that is scoped to specific capabilities.
+This guide covers the `getSessionSigs` function from the Lit SDK. For an overview of what session signatures are and how they are to be used, please go [here](./intro).
 
-In order to generate `SessionSigs`, you need a `capacityDelegationAuthSig` in signatures. This is because capacity credits are now required on both `Habanero` and `Manzano` networks. You can buy or [mint](../../capacity-credits.md#minting-capacity-credits) capacity credits centrally and [delegate](../../capacity-credits.md#delegating-access-to-your-capacity-credits-nft) them to your users. 
+The function uses the session keypair generated by the `LitNodeClient` to invoke a callback function that signs the generated session key, creating an `AuthSig` that is scoped to specific capabilities.
 
-You can generate a `SessionSig` with the help of `capacityDelegationAuthSig` object in the following way:
+Using the `getSessionSigs` function, you can specify the abilities of your current session on the Lit network. This function is the simplest way to get session signatures, at the minimum only requiring an `ethers.Wallet` and the `LitNodeClient`. It will enable your `ethers.Wallet` to perform abilities to the resources you specify in the `AuthSig`.
```

Comment:
```suggestion
Using the `getSessionSigs` function, you can specify the Lit capabilities of your current session on the Lit network. This function is the simplest way to get session signatures, at the minimum only requiring an `ethers.Wallet` and the `LitNodeClient`. It will enable your `ethers.Wallet` to perform abilities to the resources you specify in the `AuthSig`.
```